client/src/components/Home Page/PulloutMenu.tsx

```diff
@@ -169,11 +169,16 @@ const PulloutMenu: React.FC<Props> = ({ changeTheme }) => {
   return (
     <div>
       <CssBaseline />
+<<<<<<< HEAD
```

Comment:
you've committed a merge conflict here - make sure to handle all conflicts before merging this into main